python-backend/tests/auth/test_expected_auth.py

```diff
@@ -44,14 +48,21 @@
      (ExpectedDocumentUploadResource, "delete", [MINE_CREATE, MINESPACE_PROPONENT]),
      (ExpectedMineDocumentResource, "get", [MINE_VIEW]),
      (ExpectedMineDocumentResource, "post", [MINE_CREATE]),
+     (VarianceDocumentResource, "get", [MINE_VIEW]),
+     (VarianceDocumentUploadResource, "post", [MINE_CREATE, MINESPACE_PROPONENT]),
+     (VarianceDocumentUploadResource, "put", [MINE_CREATE, MINESPACE_PROPONENT]),
+     (VarianceDocumentUploadResource, "delete", [MINE_CREATE, MINESPACE_PROPONENT]),
```

Comment:
These endpoints are in (mostly) alphabetical order, could you move these to the end of the list?

Reply:
Sure. I couldn't tell if there was an order. I'll gladly move mine to the correct places